bucket/bucket.go

```diff
@@ -50,6 +51,20 @@ func (b *Bucket) GetObjectReader(key string, opts ...option.GetObjectInput) (io.
 	return resp.Body, nil
 }
 
+// GetObjectRequest generates a "aws/request.Request" representing the client's request for the GetObject operation.
```

Comment:
This also means that the test for `GetObjectRequest` will be covered by the test for `GetObject` alone. I can remove the [newly added test too](https://github.com/shihanng/aws-go-s3/blob/a586dad0c5ad28da1e2b822e1d2c7fd763ad201c/bucket/bucket_test.go#L87-L99). What do you think?